notebook/static/notebook/js/notebook.js

```diff
@@ -840,6 +840,53 @@ define(function (require) {
     // Cell movement
 
     /**
+     * Move the current selection up, keeping the same cells selected
+     * No op if the selection is at the beginning of the notebook
+     */
+    Notebook.prototype.move_selection_up = function(){
+        // actually will move the cell before the selection, after the selection
+        var indices = this.get_selected_cells_indices();
+        var first = indices[0];
+        var last = indices[indices.length-1];
+
+        var i1 = this.get_selected_index();
+        var i2 = this.get_anchor_index();
```

Comment:
Can these variable names be a little bit more descriptive? Currently, the code is a little bit hard to read.